python/lib/modeldata/dmod/modeldata/data/dataset.py

```diff
@@ -568,7 +578,8 @@ def add_data(self, dataset_name: str, **kwargs) -> bool:
         pass
 
     @abstractmethod
-    def create(self, name: str, category: DataCategory, domain: DataDomain, is_read_only: bool, **kwargs) -> Dataset:
+    def create(self, name: str, category: DataCategory, domain: DataDomain, is_read_only: bool,
```

Comment:
Yeah, I think so. I wanted it to accept arbitrary keyword args, so implementations could introduce specific params to customize behavior and still match the abstract signature. I think I remember IDE/linting complaints if I didn't put it there.

Reply:
My point was that the `**kwargs` got removed in this diff...

Reply:
Ok, time for my brain to go into 4-D chess mode ...

Right, sorry, had that backwards.  I did want to do what I described before in a lot of instances.  But there were also a few where it didn't make sense (at least initially), because doing that prevented developing to an interface, and we aren't always going to know what kind of dataset manager or dataset we are dealing with.

I _think_ this was one of those times.  Although also it's possible that I changed things back in a later branch.